ingestion/src/metadata/ingestion/source/database/dbt_source.py

```diff
@@ -96,14 +97,25 @@ def _parse_data_model(self):
                     schema = mnode["schema"] if mnode["schema"] else "default"
                     raw_sql = mnode.get("raw_sql", "")
                     description = mnode.get("description")
-                    dbt_user_name = cnode["metadata"].get("owner")
-                    user_name = f"*{dbt_user_name}*"
-                    user_fqn = fqn.build(
-                        self.metadata, entity_type=User, user_name=user_name
-                    )
-                    owner = self.metadata.get_entity_reference(
-                        entity=User, fqn=user_fqn
-                    )
+                    dbt_owner = cnode["metadata"].get("owner")
+                    owner = None
+                    if dbt_owner:
+                        owner_name = f"*{dbt_owner}*"
+                        user_owner_fqn = fqn.build(
+                            self.metadata, entity_type=User, user_name=owner_name
+                        )
+                        if user_owner_fqn:
+                            owner = self.metadata.get_entity_reference(
+                                entity=User, fqn=user_owner_fqn
+                            )
+                        else:
+                            team_owner_fqn = fqn.build(
+                                self.metadata, entity_type=Team, team_name=owner_name
+                            )
+                            if team_owner_fqn:
+                                owner = self.metadata.get_entity_reference(
+                                    entity=Team, fqn=team_owner_fqn
+                                )
```

Comment:
Let's give them an else warning the specified user doesn't exists in OpenMetadata..
if user, else if team, else doesn't exist warning..

Reply:
Got it I'll add the warning @ayush-shah

Reply:
@ayush-shah added the warning